### ISSUE TITLE
Add a script to compare models

### ISSUE THREAD

**Issue by dthaler** [dthaler/pods-ai#126]
run_inference.py can run a single sample against a specified model and report the results

But we have three models to evaluate: fastai, orcahello, and huggingface.
We should have a script that can compare the results across the three models using a test set.

Currently training_samples.csv contains both the training set and the validation set,
so we can treat rows in detections.csv that have no corresponding row in training_samples.csv. as a potential test set.

Evaluation should be done by running each model against the same test set of 60-second samples,
and reporting the number or percentage of correct identifications, false positives, and false negatives.

One wrinkle is that fastai and orcahello have only 2 classes whereas huggingface has 7.
As a result, fastai and orcahello only detect "resident" vs other (everything else), whereas hugging face detects "resident" vs specific other classes.

Thus a "correct" identification means resident when resident was expected, or anything other than resident when anything other than resident was expected, regardless of whether the specific class was correct.
A "false positive" means the model reported resident when the correct class was something else.
A "false negative" means the model reported something other than resident, when the correct class was resident.
